instrumentation/dubbo/dubbo-apache-2.7/javaagent/dubbo-apache-2.7-javaagent.gradle

```diff
@@ -0,0 +1,17 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
```

Comment:
Can we name the folders either `apache-dubbo/apache-dubbo-2.7` for consistency with other apache projects, or just `dubbo/dubbo-2.7`? I guess dubbo is a unique enough name that apache may not be needed but whatever seems more familiar to users I guess

Reply:
In China, Alibaba's Dubbo still has a certain number of users

Reply:
Ah got it - then yeah let's use `apache-dubbo/apache-dubbo-2.7`

Reply:
Maybe it would be more cautious to separate Alibaba Dubbo and Apache Dubbo

Reply:
Ok I got it